controllers/cruisecontroloperation_controller.go

```diff
@@ -144,7 +153,7 @@ func (r *CruiseControlOperationReconciler) Reconcile(ctx context.Context, reques
 	}
 
 	// Checking Cruise Control health
-	status, err := r.scaler.Status(ctx)
+	status, err := r.getStatus(ctx, log, kafkaCluster, kafkaClusterRef, ccOperationListClusterWide)
```

Comment:
we dont need to pass the kafkaClusterRef because it is the reference of the kafkaCluster what is also passed as argument (https://github.com/banzaicloud/koperator/blob/782996952d7af9886b7acfd67ea9f386f116aa6f/controllers/cruisecontroloperation_controller.go#L130)

Reply:
Changed in https://github.com/banzaicloud/koperator/pull/974/commits/2c47d7dbbae5a00806acb152d172b3ad4ada2b42